### PR TITLE
[sogo] added hooks support

### PR DESCRIPTION
**Pull request by feldsam**
Signed-off-by: Kristian Feldsam <feldsam@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 !data/conf/nginx/dynmaps.conf
 !data/conf/nginx/meta_exporter.conf
 !data/conf/nginx/site.conf
+!/**/.gitkeep
```
Comment (andryyy) [inline .gitignore:4]: Is this related?

Comment (andryyy) [reply .gitignore:4]: Ah, forget it. :)

**Comment by feldsam** [comment]
@andryyy what about this PR?

**Comment by andryyy** [comment]
We really need to add docs for it. :(
